web-client/public/js/graph.js

```diff
@@ -1105,40 +1106,34 @@ export var drawGraph = function (network, nodeColoring) {
         }
     };
 
-    nodeColoring.removeNodeColoring = function () {
-        grnState.nodeColoring.nodeColoringEnabled = false;
-        node.selectAll(".coloring").remove();
-    };
-
     updaters.removeNodeColoring = function () {
         grnState.nodeColoring.nodeColoringEnabled = false;
         node.selectAll(".coloring").remove();
     };
 
     updaters.renderNodeColoring = function () {
         if (grnState.nodeColoring.nodeColoringEnabled) {
-            colorNodes("top", grnState.nodeColoring.topDataset, grnState.nodeColoring.avgTopDataset,
+            colorNodes("top", grnState.nodeColoring.topDataset, grnState.nodeColoring.averageTopDataset,
                         grnState.nodeColoring.logFoldChangeMaxValue);
-            colorNodes("bottom", grnState.nodeColoring.bottomDataset, grnState.nodeColoring.avgBottomDataset,
+            colorNodes("bottom", grnState.nodeColoring.bottomDataset, grnState.nodeColoring.averageBottomDataset,
                         grnState.nodeColoring.logFoldChangeMaxValue);
             renderNodeLabels();
             renderNodeColoringLegend(grnState.nodeColoring.logFoldChangeMaxValue);
         }
     };
 
-    nodeColoring.renderNodeColoring = function () {
-        if (grnState.nodeColoring.nodeColoringEnabled) {
-            colorNodes("top", grnState.nodeColoring.topDataset, grnState.nodeColoring.avgTopDataset,
-                        grnState.nodeColoring.logFoldChangeMaxValue);
-            colorNodes("bottom", grnState.nodeColoring.bottomDataset, grnState.nodeColoring.avgBottomDataset,
-                        grnState.nodeColoring.logFoldChangeMaxValue);
-            renderNodeLabels();
-            renderNodeColoringLegend(grnState.nodeColoring.logFoldChangeMaxValue);
+    const hasExpressionData = (sheets) => {
```

Comment:
Small ES6 formatting note: when an arrow function has a single argument, the parentheses are optional.
```suggestion
    const hasExpressionData = sheets => {
```